plugin/src/test/groovy/com/novoda/test/TestProject.groovy

```diff
@@ -32,8 +32,11 @@ ${project.additionalConfiguration}
         this.gradleRunner = GradleRunner.create()
                 .withProjectDir(projectDir)
                 .withPluginClasspath()
+                .withTestKitDir(new File(projectDir, "test-kit"))
```

Comment:
@pablisco what is this fixing?

Reply:
This is for the problem of locking files while doing the internal build. Otherwise it shares the test-kit files and none of the tests work